modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/new-js/components/field-sets/FieldSetModal.scss

```
@@ -1,15 +1,17 @@
-.fieldset-modal {
-	.data-engine-form-builder > .sheet {
-		background-color: transparent;
-		border: 0 none;
-	}
+$multiPanelSidebarContentWidth: 368px;
```

Import `_vatiables.scss` to get access to `$multiPanelSidebarContentWidth`

```suggestion
@import '../../../css/variables';
```

---

Hi @miltonmc. The variable css is in another module, so I don't have access to get it